src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2616,4 +2607,32 @@ public function getNamespace(): string
     {
         return 'http://symfony.com/schema/dic/symfony';
     }
+
+    protected function isConfigPathEnabled(string $path): bool
+    {
+        if (isset($this->configsEnabled[$path])) {
+            return $this->configsEnabled[$path];
+        }
+
+        throw new LogicException('Can not read config enabled because it has not been initialized.');
+    }
+
+    protected function readConfigEnabled(ContainerBuilder $container, array $config, string $path): bool
```

Comment:
why protected ? (same above)

also, let's make $path the first argument for all methods, it will make it easier to visually parse the code I think